File: Daily Pratice Question/C LANGUAGE/Question(24).c
// 24. Write a program to remove whitespace from both ends of a string.
// Example: Input: " hello ", Output: "hello".



#include <stdio.h>
#include<string.h>

int main() {
   char str[]=" h ell o ";
  int i;
  for(i=0;i<strlen(str);i++){
      if(str[i]!=32){
          printf("%c",str[i]);
      }
  }
    return 0;
}

File: Daily Pratice Question/JAVASCRIPT/question(24).js
// 24. Write a program to remove whitespace from both ends of a string.
// Example: Input: " hello ", Output: "hello".


var arr="h  e l lo"
 var removespace=''

for(i=0;i<arr.length;i++){
    if(arr[i]!==' '){
        removespace+=arr[i]
        
    }
}console.log(removespace)



